### PR TITLE
Billing new plans

### PR DESCRIPTION
**Pull request by MichaelMacaulay**

### CODE REVIEW

File: website/pages/en/billing.mdx [hunk 1]
```diff
@@ -2,104 +2,84 @@
 title: Billing
 ---
 
-> Invoices are generated on a weekly basis.
+## Subgraph Billing Plans
 
-There are two options for paying for your query fees:
+There are two plans to use when querying subgraphs on The Graph Network. You can learn more about these plans on our Subgraph Pricing Page (coming soon).
```
Comment (azf20) [inline website/pages/en/billing.mdx:7]: Not sure it makes sense to tease the Pricing page? Also are we calling it the Subgraph Pricing Page?

File: website/pages/en/billing.mdx [hunk 2]
```diff
@@ -2,104 +2,84 @@
...
-3. Click the dropdown next to the 'Add GRT' button at the center of the page. Select withdraw GRT. A side panel will appear.
-
+3. Click the "Manage"" button at the top right corner of the page. Select "Withdraw GRT". A side panel will appear.
 4. Enter the amount of GRT you would like to withdraw.
-
 5. Click 'Withdraw GRT' to withdraw the GRT from your account balance. Sign the associated transaction in your wallet. This will cost gas. The GRT will be sent to your Arbitrum wallet.
-
 6. Once the transaction is confirmed, you'll see the GRT withdrawn from your account balance in your Arbitrum wallet.
 
 ### Adding GRT using a multisig wallet
```
Comment (azf20) [inline website/pages/en/billing.mdx:55]: Is this process different except that you need to connect a multisig?

**Review by azf20 — CHANGES_REQUESTED** [review]
Small comment on pricing page mention

File: website/pages/en/billing.mdx (hunk 1 above)
Comment (MichaelMacaulay) [reply website/pages/en/billing.mdx:7]: Removed!

File: website/pages/en/billing.mdx (hunk 2 above)
Comment (MichaelMacaulay) [reply website/pages/en/billing.mdx:55]: Removed that, thanks for flagging!

**Pull request merged**